torch/_torch_docs.py

```diff
@@ -718,7 +718,7 @@ def merge_dicts(*dicts):
     \text{out}_{i} = \tanh^{-1}(\text{input}_{i})
 """ + r"""
 Args:
-    {input}: The domain is the open interval `(-1, 1)`.
+    input (Tensor): the input tensor where math:`input` is in the open interval `(-1, 1)`.
```

Comment:
Added a statement here as well for the input argument. But this is open for discussion, do we need a statement here?

I ask this question beause it doesn't throw a `RuntimeError`, instead returns `nan` and `inf`. NumPy raises a `RuntimeWarning` instead of an error, and returns `nan` as well. This pattern comes directly from `std::op`, I think. Taking a look at `torch.acos` [documentation](https://pytorch.org/docs/master/generated/torch.acos.html) - it doesn't have a statement as well while it's domain is `[-1, 1]` (for other values, it returns `nan`).

One possible solution is, we can say in the function documentation: `All values in the input tensor should be in the open interval (-1, 1)` instead of `All values in the input tensor must be in the open interval (-1, 1)`. I'm not too sure, and will appreciate your take on this.

cc: @mruberry, @albanD

Reply:
Great follow-up. You're right, I mistakenly said that NumPy threw an error previously and I was wrong.

I think we can just say be simple and say what happens. Something like, "values outside the function's domain return nan."

Reply:
Thanks @mruberry for your review. :) I've made the commits, and will be adding a few follow-up questions (should be a final round, but just want to be thorough before this PR lands to upstream).